java-client/src/generated/java/org/opensearch/client/opensearch/indices/ExistsAliasRequest.java

```diff
@@ -46,238 +53,312 @@
 import org.opensearch.client.transport.endpoints.BooleanResponse;
 import org.opensearch.client.transport.endpoints.SimpleEndpoint;
 import org.opensearch.client.util.ApiTypeHelper;
+import org.opensearch.client.util.CopyableBuilder;
 import org.opensearch.client.util.ObjectBuilder;
 import org.opensearch.client.util.ObjectBuilderBase;
+import org.opensearch.client.util.ToCopyableBuilder;
 
 // typedef: indices.exists_alias.Request
 
 /**
  * Returns information about whether a particular alias exists.
- *
  */
+@Generated("org.opensearch.client.codegen.CodeGenerator")
+public class ExistsAliasRequest extends RequestBase implements ToCopyableBuilder<ExistsAliasRequest.Builder, ExistsAliasRequest> {
 
-public class ExistsAliasRequest extends RequestBase {
     @Nullable
     private final Boolean allowNoIndices;
 
+    @Nonnull
     private final List<ExpandWildcard> expandWildcards;
 
     @Nullable
     private final Boolean ignoreUnavailable;
 
+    @Nonnull
     private final List<String> index;
```

Comment:
This seems to be also not in sync? Shouldn't be `indices`? https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/action/admin/indices/alias/get/GetAliasesRequest.java#L53